site/tool_configuration.md

```diff
@@ -1,10 +1,26 @@
 ## Tools Configuration
+ - [Tool Property File](#tool-properties)
  - [Model Filters](#model-filters)
  - [Target Environments](config/target_env.md)
  - [Type Definitions](#domain-type-definitions)
  - [Variable Injection](variable_injection.md)
  - [Custom Configuration](#custom-configuration)
 
+ ### Tool Properties
+ The WebLogic Deploy Tooling tools can be configured or tuned from the tool property file. This property file is installed as `<weblogic-deploy>/lib/tool.properties`. You may tune any of the properties in this file or override the file completely by providing your own tool.properties file. Reference the directory where the tool.properties resides using the  `CUSTOM_CONFIG_VARIABLE` Environment variable.
```

Comment:
The environment variable is WDT_CUSTOM_CONFIG. Other sections have this blurb:
"Another option is to configure . . . in a [Custom Configuration](#custom-configuration) directory. Create the `. . .` file in the `$WDT_CUSTOM_CONFIG` directory."

Also add Tools Configuration to the list in the Custom Configurations section.